compiled_starters/clojure/build.clj

```diff
@@ -0,0 +1,22 @@
+(ns build
+  (:gen-class)
+  (:require [clojure.tools.build.api :as b]))
+
+(def lib 'io.codecrafters.redis)
+(def class-dir "/tmp/codecrafters-build-redis-clojure/classes")
+(def basis (b/create-basis {:project "deps.edn"}))
+(def uber-file "/tmp/codecrafters-build-redis-clojure/redis.jar")
```

Comment:
```suggestion
(def uber-file "/tmp/codecrafters-build-redis-clojure/target.jar")
```

(Since redis is already in the file path, let's try making this less variable?)